city_scrapers/spiders/ind_indygo.py

```diff
@@ -17,17 +17,28 @@ def parse(self, response):
         Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
         needs.
         """
-        for item in response.css(".content-section:nth-child(4) ul li"):
+        meeting_section = response.css(".content-section:nth-child(4)")
+        section_title = meeting_section.css("h2::text").get()
+        meeting_year = section_title.split(" ")[0]
+        meeting_time = (
+            (response.css(".content-section:nth-child(4) p strong::text").get())
+            .split(":", 1)[1]
+            .split("EST")[0]
+            .strip()
+        )
+        print(meeting_time)
+
+        for date_item in meeting_section.css("ul li"):
             meeting = Meeting(
-                title=self._parse_title(item),
-                description=self._parse_description(item),
-                classification=self._parse_classification(item),
-                start=self._parse_start(item),
-                end=self._parse_end(item),
-                all_day=self._parse_all_day(item),
-                time_notes=self._parse_time_notes(item),
-                location=self._parse_location(item),
-                links=self._parse_links(item),
+                title="IndyGo Board",
+                description=self._parse_description(date_item),
```

Comment:
Since we're not doing any actual parsing here, lets delete the _parse_description method entirely and simply set description's value to "" here. Eg.

```
description=""
```
These methods are simply helpers from the spider template and there's nothing wrong with deleting them entirely or changing them based on a spider's needs. Removing parse methods where they're unused makes the code easier to understand for other contributors. I'd be appreciative if you could review other areas of your code (including the surrounding lines of this file) where this would apply.